courses/free-courses-hi.md

```diff
@@ -48,6 +50,11 @@
 
 *[DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
 
+### Git and GitHub
+
+* [Complete Git Tutorials For Beginners In Hindi playlist](https://youtube.com/playlist?list=PLu0W_9lII9agwhy658ZPA0MTStKUJTWPi) - CodeWithHarry
+* [Git & GitHub Tutorial For Beginners In Hindi](https://www.youtube.com/watch?v=gwWKnnCMQ5c) - CodeWithHarry
```

Comment:
is there a playlist?

Reply:
Yeah some, do I have to mention it differently?

Reply:
✔️ 👍 1h15min video with timeline chapters